rules/macos/execution_curl_spawned_from_installer_package.toml

```diff
@@ -0,0 +1,71 @@
+[metadata]
+creation_date = "2021/02/23"
+maturity = "production"
+updated_date = "2021/02/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule identifies when the built in macOS Installer program is used to install a package that spawns a shell and runs
+the cURL command.
```

Comment:
```suggestion
Identifies when the built in macOS Installer program is used to install a package that spawns a shell and runs
the cURL command. This activity has been observed being leveraged by malware.
```

Will need a re-lint

Reply:
if you want to mention the `Silver Sparrow` campaign here in the description, instead of the note section, that would work.